apps/console/src/app/components/prompts/views/PromptEditView.tsx

```diff
@@ -28,12 +28,13 @@ export const PromptEditView = () => {
   const {
     form,
     handleFormValuesChange,
-    isChangesToCommit,
+
     variables,
     setVariable,
   } = usePromptEdit();
   const { prompt, currentPromptVersion, integration, isDraft } =
     useCurrentPrompt();
+
```

Comment:
Let's delete this one, and rename the new `PromptPlaygroundView` to `PromptEditView`